### PR TITLE
Add initial code owners configuration

### PR DESCRIPTION
**Pull request by swissspidy**
Moving checks into separate folders per category makes it easy to set up code owners.

Feel free to submit suggestions to the CODEOWNERS file if you like to be included/excluded.

See #448

### CODE REVIEW

**Review by joemcgill — COMMENTED** [review]
I think this is a great idea. To make management easier, we could consider assigning teams like @WordPress/performance-admins to the Performance section. Not sure if other community teams have similar groups that could be used.

Some practical questions/concerns:

1. This should include a changelog entry, since several namespaces are changing.
2. There a chance that someone is already extending this plugin with their own custom checks. Do we need to be concerned about adding backcompat support for the previous  names, or is adding documentation to the release notes enough?

**Comment by swissspidy** [comment]
We can create new teams for each category or something if we want, e.g. `@WordPress/plugin-check-plugins-team`, `@WordPress/plugin-check-performance`. Don't want to use that existing team though, as it includes too many people who aren't contributing this plugin and would just unnecessarily get pinged because of it (like right now :)).

(Edit: also keep in mind mentioning/assigning teams doesn't work for people not in the org or when logged out of GitHub)

Not worried about back compat, a changelog entry should be enough.

**Review by westonruter — APPROVED** [review]

**Review by joemcgill — APPROVED** [review]

File: readme.txt [hunk 1]
```diff
@@ -68,9 +68,10 @@ In any case, passing the checks in this tool likely helps to achieve a smooth pl
 
 == Changelog ==
 
-= 1.0.2 =
+= 1.1.0 =
 
 * Fix - Correct detection of readme files in Windows by normalizing file paths.
+* Enhancement - Changed the namespace of included checks. [#460](https://github.com/WordPress/plugin-check/pull/460)
```
Comment (mukeshpanchal27) [inline readme.txt]:
```suggestion
* Enhancement - Changed the namespace of included checks. ([460](https://github.com/WordPress/plugin-check/pull/460))
```
Not blocker: In PL we use above formate for PR links.

**Review by mukeshpanchal27 — COMMENTED** [review]
Thanks, @swissspidy, for the PR. Overall, it looks good to me. I've left one nit-pick and a couple of questions:

- Who is responsible for the other checks and other infrastructure-related files?
- The documentation at https://github.com/WordPress/plugin-check/tree/trunk/docs should be updated to reflect the new changes related to checks.

File: readme.txt (hunk 1 above)
Comment (swissspidy) [reply readme.txt]: Here we don‘t use that format, see older changelog entries

**Comment by swissspidy** [comment]
> Who is responsible for the other checks and other infrastructure-related files?

Everyone and no one :) Something we can define as we go. Suggestions welcome if you think we should define all of that already.

**Comment by adamsilverstein** [comment]
Looks great and this will let individual teams iterate more rapidly. +1.

**Review by mukeshpanchal27 — APPROVED** [review]

**Comment by swissspidy** [comment]
@bordoni @chriscct7 @davidperezgar @barrykooij Would appreciate eyes on this from the plugins team as well as it means you'll get notified more often for PRs (every time some touches checks related to your team).

IMHO adding a code owners configuration is a great way to more clearly separate concerns. Even if we end up restructuring the plugin further down the road, this would be a good thing to do.

**Comment by davidperezgar** [comment]
I wait @barrykooij have an opinion of this.

But it looks great as we have very clear information where to see the checks.

**Review by davidperezgar — APPROVED** [review]

**Pull request merged**